server/graphql/general/src/queries/me.rs

```diff
@@ -25,7 +26,13 @@ pub fn me(ctx: &Context<'_>) -> Result<UserResponse> {
     let service_provider = ctx.service_provider();
     let service_ctx = service_provider.context()?;
     let user_service = UserAccountService::new(&service_ctx.connection);
-    let user = match user_service.find_user(&user.user_id) {
+
+    let key_value_store = KeyValueStoreRepository::new(&service_ctx.connection);
+    let site_id = key_value_store
+        .get_i32(KeyValueType::SettingsSyncSiteId)?
+        .unwrap();
```

Comment:
You can move this "logic" part into the service find_user method, see comment below.